src/ChainTransfer/ChainTransferBase.cs

```diff
@@ -18,134 +18,136 @@ public bool Active
 			else Down();
```

Comment:
Not directly related to this PR, but the methods in the setter are being called before the scene is ready, so the children the methods referencing don't exist yet, and we get errors.

Reply:
Fixed.